speedt/speedt.go

```diff
@@ -156,6 +150,6 @@ func Run(nodeCN *mongo.Client, stg storage.Storage, compression pbm.CompressionT
 // without allocations
 func StringToBytes(s string) []byte {
 	sh := (*reflect.StringHeader)(unsafe.Pointer(&s))
-	bh := reflect.SliceHeader{sh.Data, sh.Len, sh.Len}
-	return *(*[]byte)(unsafe.Pointer(&bh))
+	bh := &reflect.SliceHeader{sh.Data, sh.Len, sh.Len}
+	return *(*[]byte)(unsafe.Pointer(bh))
```

Comment:
Please, revert these changes. Unless you are sure it fixes something. Maybe these are ok, but I'm not sure if initializing `bf` as a point won't make its escape.